prac_05/hex_colours.py

```diff
@@ -0,0 +1,26 @@
+
+"""look up hex colours"""
+
+"""constant dictionary"""
+
+HEX_COLOURS = {"aliceblue": "#f0f8ff", "antiquewhite": "#faebd7",
+                "antiquewhite1": "#ffefdb", "antiquewhite2": "#eedfcc",
+                "antiquewhite3": "#cdc0b0", "antiquewhite4": "#8b8378",
+                "aquamarine1": "#7fffd4", "aquamarine2": "#76eec6",
+                "aquamarine4": "#458b74", "azure1": "#f0ffff",
+                "azure2": "#e0eeee", "azure3": "#c1cdcd", "azure4": "#838b8b",
+                "beige": "#f5f5dc", "bisque1": "#ffe4c4"}
+
+"""Main"""
+
+def main():
+    colour = input("Enter short colour: ").lower()   #<--- use .upper() to change to all entries to capital
```

Comment:
What is a "short" colour? Ambiguous prompt may illicit the wrong input from the user (nitpick). Otherwise well written.